data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.data.backends.gae.api
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to the Oppia platform parameter endpoint. */
+interface PlatformParameterService {
+
+  /**
+   * This function returns a Map of Platform Parameters where the keys corresponds to parameter
+   * names and values corresponds to their server value.
+   * [Reference](https://github.com/oppia/oppia/blob/develop/core/controllers/platform_feature.py)
```

Comment:
Can you pin this to a specific commit hash rather than referencing develop? If the file moves or is renamed in the future, this doc string will become invalid.

Reply:
Done. Pointing to the latest commit for this file

Reply:
Also: keep in mind you can just use the first ~6 characters of the hash to keep it shorter (to check, just navigate to the shorter URL & make sure it works correctly).